tests/Tests.elm

```diff
@@ -284,5 +284,12 @@ all =
                         Expect.equal (Just Medium) <|
                             getDifficulty (List.repeat 2 3) (List.repeat 2 100)
                 ]
```

Comment:
I missed this square bracket on this change. This line should have been deleted since I added the square bracket at this level of indentation at the end of my changes, to close the list of tests.